### PR TITLE
revamp modal close button

### PR DESCRIPTION
**Pull request by itkg-ngilain**
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1231
https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/88

### CODE REVIEW

**Comment by itkg-canne** [comment]
:+1:

File: BackofficeBundle/Resources/views/BackOffice/Underscore/orchestraModalView._tpl.twig [hunk 1]
```diff
@@ -1,9 +1,8 @@
 <div class="modal-dialog">
     <div class="modal-content">
         <div class="modal-header clearfix">
-            <button type="button" class="close" data-dismiss="modal">
-                <span aria-hidden="true">&times;</span>
-                <span class="sr-only">{{ ('open_orchestra_backoffice.modal.close') | trans() }}</span>
+            <button type="button" class="btn btn-success" data-dismiss="modal">
```
Comment (nicolasThal) [inline BackofficeBundle/Resources/views/BackOffice/Underscore/orchestraModalView._tpl.twig]: I would rather put btn-warning instead of success.

**Comment by itkg-ngilain** [comment]
@nicolasThal : update done

**Comment by itkg-nanne** [comment]
:+1:

**Pull request merged**